tests/VolleyManagement.UnitTests/Services/GameService/GameServiceTests.cs

```diff
@@ -1648,6 +1648,80 @@ public void Edit_GameValid_GameEdited()
             VerifyEditGame(game, Times.Once());
         }
 
+        /// <summary>
+        /// Test for Edit method. Change time for game in second round with unknown teams. 
+        /// No exceprtion returns and game is edited successfully.
+        /// </summary>
+        [TestMethod]
+        public void Edit_ChangeTimeForPlayOffGameWithUnknownTeams_GameSaved()
+        {
+            //Arrange
+            bool isExceprionThrown = false;
+            var testTournament = CreatePlayoffTournament();
+            var games = new GameTestFixture()
+                .TestEmptyGamePlayoffSchedule()
+                .Build();
+
+            var testGameForEdit = games[5];
+            testGameForEdit.GameDate = new DateTime(2016, 4, 3, 0, 0, 0);
```

Comment:
Does  this date mean anything?

Reply:
Created tournament has start and end date. That's why I chose game time in this special range.